cypress/support/api.commands.js

```diff
@@ -57,3 +57,39 @@ Cypress.Commands.add('contentCreation', (body) => {
    })      
 
 });
+
+Cypress.Commands.add('upGradeDownGradeUsersubscription', (tier_type) => {
+  let new_package_id = null;
+  let new_subscription_id = null;
+
+  cy.fixture('package_ids.json').then((data) => {
+    if (tier_type === 'tier_1') {
+      new_package_id = data.tier_1_package_id; 
+    } else if (tier_type ==='tier_2'){
+      new_package_id = data.tier_2_package_id; 
+    } else if (tier_type ==='tier_free'){
+      new_package_id = data.tier_free_package_id; 
+    } else {
+     new_package_id = tier_type;
+    cy.log('No such tier_type like that! Please try "tier_1, Tier_2 or tier_free') 
+    }
+      
```

Comment:
What about throwing error instead of cy.log(). And maybe use switch instead of if-else?

Reply:
done